src/main/java/org/breedinginsight/brapi/v2/dao/BrAPIGermplasmDAO.java

```diff
@@ -142,8 +142,11 @@ private void setup() {
                 List<String> parents = Arrays.asList(germplasm.getPedigree().split("/"));
                 if (parents.size() >= 1) {
                     if (programGermplasmByFullName.containsKey(parents.get(0))) {
-                        //problem, if parent not in germplasmbyfullname, not replaced by accession number, assumption that parents are also in file
                         newPedigreeString = programGermplasmByFullName.get(parents.get(0)).getAccessionNumber();
+                    } else if (parents.get(0).equals("NA")){
+                        //Handle NA values from breedbase, may be modified later to distinguish between unknown vs null
+                        newPedigreeString = null;
+                        parents = new ArrayList<>();
```

Comment:
I think this fix also needs to be applied to the block below if there are two parents (starts on line 152)